### PR TITLE
New issue: Bogus postcondition for filesystem_error constructor

### PR DESCRIPTION
**Pull request by timsong-cpp**

### CODE REVIEW

File: xml/issue3043.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<title>Bogus postcondition for <tt>filesystem_error</tt> constructor</title>
+<section><sref ref="[fs.filesystem_error.members]"/></section>
+<submitter>Tim Song</submitter>
+<date>6 Dec 2017</date>
+<priority>99</priority>
+
+<discussion>
+<p>
+<sref ref="[fs.filesystem_error.members]"/> says that constructors of
+<tt>filesystem_error</tt> has the postcondition <tt>runtime_error::what()</tt> has the
```
Comment (CaseyCarter) [inline xml/issue3043.xml]: "constructors...has" should be "have"

This sentence would also be easier to parse with quotes around the quoted text.

Comment (timsong-cpp) [reply xml/issue3043.xml]: It's a table, not exactly a quote. Will fix grammar in a sec.

Comment (CaseyCarter) [reply xml/issue3043.xml]: Clarifying:

> ...constructors of <tt>filesystem_error</tt> have the postcondition <tt>runtime_error::what()</tt> has the value <tt>what_arg.c_str()</tt>.

reads as "foo have the bar has the baz." It needs quotes:

> ...constructors of <tt>filesystem_error</tt> have the postcondition "<tt>runtime_error::what()</tt> has the value <tt>what_arg.c_str()</tt>".

Comment (timsong-cpp) [reply xml/issue3043.xml]: I added an extra "that" after "postcondition" instead.

**Pull request merged**